docs/en/workshop/background.md

```diff
@@ -3,27 +3,32 @@ title: Quantum Computing For Drug Discovery Workshop
 chapter: true
 ---
 
-# Quantum Computing For Drug Discovery Solution
+# Quantum Ready Architecture For Drug Discovery
 
-Quantum Computing For Drug Discovery Solution is a cloud-native solution for the application of quantum computing
+Quantum Ready Architecture For Drug Discovery is a cloud-native solution for the application of quantum computing
 technology for drug discovery. This solution is trying to bridge the gap between the industrial
 knowledge and the novel computation model provided by the quantum technologies. This solution will continually 
 implement the Proof of Concepts (PoC) based on the recent publications or reports. It is suitable for customers
 with needs to evaluate the possible useful cases of quantum computing in their industries. No matter you just
 start to consider this new technology or on the way, you can use this solution to speed up the process, save
 time and cost. It is an open-source projects, anyone can contribute to features of adding more related solutions.
 
-![Application of Quantum Computing](../images/application-of-qc.png)
-
+Quantum computing have many use cases, please refer to this 
```

Comment:
delete the image `application-of-qc.png` from source as well

Reply:
done